fcrepo-http-api/src/main/resources/views/common-node-actions.vsl

```diff
@@ -86,20 +83,32 @@ WHERE { }
     <hr />
 </form>
 
+#if ( !$isRoot )
+	#if ( $isLDPNR )
+	<h3>Versioning</h3>
+	<p>Versioning of binary resources is not currently available through the HTML UI.</p>
+	<hr />
+	#elseif ($isVersionable == true)
 <form id="action_create_version" name="action_create_version" method="POST">
     <h3>Create Version Snapshot</h3>
-    <div class="form-group">
-    <input type="text" id="version_id" placeholder="(auto-generated name)" name="version_name" class="form-control"/>
-    </div>
     <button id="create-version-button" type="submit" class="btn btn-primary">Create Version</button>
+    <h4><a href="$topic/fcr:versions">View Mementos</a></h4>
     <hr />
 </form>
+	#else
+<form id="action_enable_version" name="action_enable_version" method="POST">
+    <h3>Versioning</h3>
+    <button id="enable-version-button" type="submit" class="btn btn-primary">Enable Versioning</button>
+    <hr />
+</form>
+#end
 
 <form id="action_delete" name="action_delete">
     <h3>Delete Resource</h3>
     <button name="delete-button" type="submit" class="btn btn-danger">Delete</button>
     <hr />
 </form>
+	#end
```

Comment:
is this indent correct?

Reply:
I think so, but I didn't indent the actual HTML just the Velocity template tags. I can move them around if it helps, the `#end` on line 104 should be indented as well.

Reply:
I'm not familiar with Velocity at all, so whatever makes the most sense.